tensorflow/core/common_runtime/gpu/gpu_device.cc

```diff
@@ -1316,7 +1283,9 @@ std::vector<CudaVersion> GetSupportedCudaComputeCapabilities() {
 #undef TF_XSTRING
   auto extra_capabilities = str_util::Split(extra_cuda_caps, ',');
   for (const auto& capability : extra_capabilities) {
-    cuda_caps.push_back(CudaVersion(capability));
+    DeviceVersion device_version =
+        DeviceVersion::Parse(capability).ValueOrDie();
+    cuda_caps.push_back(device_version);
```

Comment:
Nit, recommend inlining the `device_version` variable:

`cuda_caps.push_back(DeviceVersion::Parse(capability).ValueOrDie())`